src/js/components/AppVersionComponent.jsx

```diff
@@ -134,6 +135,16 @@ var AppVersionComponent = React.createClass({
       ? <UnspecifiedNodeComponent />
       : <dd><pre>{JSON.stringify(appVersion.container, null, 2)}</pre></dd>;
 
+    var dependenciesNode = (appVersion.dependencies.length === 0)
+      ? <UnspecifiedNodeComponent />
+      : appVersion.dependencies.map(function (d) {
+        return (
+          <dd key={d}>
+            <Link to="app" params={{appId: encodeURIComponent(d)}}>{d}</Link>
+          </dd>
+        );
+      });
```

Comment:
Yeah, could do this in another PR. Want to be consistent with the other notations in this component.